src/main/kotlin/com/lovelysystems/gradle/LSGit.kt

```diff
@@ -97,8 +97,13 @@ class LSGit(val dir: File) {
     }
 
     fun latestLocalGitTagVersion(): Version? {
-        return gitCmd("tag", "-l", "--sort=-v:refname").lines().find { isProductionVersion(it) }?.let {
-            Version.fromIdent(it)
+        // --merged flag fails if there are no commits
+        try {
+            return gitCmd("tag", "-l", "--sort=-v:refname", "--merged").lines().find { isProductionVersion(it) }?.let {
+                Version.fromIdent(it)
+            }
+        } catch (e: Exception) {
```

Comment:
use specific excdeption

Reply:
ftr: as discussed, we will filter tags "by hand" and not use --merged, since it adds the additional requirement of having a consistent git tree